build/dependencies.yaml

```diff
@@ -226,3 +226,10 @@ dependencies:
     refPaths:
     - path: test/utils/image/manifest.go
       match: configs\[CheckMetadataConcealment\] = Config{list\.PromoterE2eRegistry, "metadata-concealment", "\d+\.\d+"}
+
+  # cadvisor 
+  - name: "gcr.io/cadvisor/cadvisor: dependents"
+    version: "v0.43.0"
+    refPaths:
+    - path: test/e2e_node/resource_collector.go
```

Comment:
can you do it for test/e2e_node/image_list.go  as well?

Reply:
Sure. Added